capstone/src/main/webapp/youtubeTypescript.js

```diff
@@ -543,3 +550,30 @@ function formatCommentForSpreadsheet(comment) {
     formattedComment = formattedComment.replace(/\s+/g, " ");
     return formattedComment;
 }
+/** Creates chart of analyzed comments and requested attributes*/
+function drawTableChart() {
+    var requestedAttributes = getRequestedAttributes();
```

Comment:
```suggestion
    const requestedAttributes = getRequestedAttributes();
```

Reply:
Since this JavaScript file is a result of compiling TypeScript I have little control over whether or not the code is style complaint. To simplify and cleanup the codebase I have added "youtubeTypescript.js" to .gitignore in "add gitignore and delete cache". In the future, youtubeTypescript.js will no longer be visible.